Polygon_mesh_processing/include/CGAL/Side_of_triangle_mesh.h

```diff
@@ -181,6 +181,25 @@ class Side_of_triangle_mesh
     box = tree.bbox();
   }
 
+  /**
+  * Constructor moving an instance of Side_of_triangle_mesh to a new memory
+  * location with minimal memory copy.
+  * @param other The instance to be moved
+  */
+  Side_of_triangle_mesh(Side_of_triangle_mesh&& other) :
+    tm_ptr{std::move(other.tm_ptr)},
+    opt_vpm{std::move(other.opt_vpm)},
+    own_tree{std::move(other.own_tree)},
+    box{std::move(other.box)},
+    #ifdef CGAL_HAS_THREADS
+      atomic_tree_ptr{other.atomic_tree_ptr.load()}
+    #else
+      tree_ptr{std::move(other.tree_ptr)}
+    #endif
+  {
+    other.own_tree = false;
+  }
+
```

Comment:
```suggestion
  Side_of_triangle_mesh(Side_of_triangle_mesh&& other)
  {
    *this = std::move(other);
  }
```

Reply:
@barcode Are you still willing to contribute this pull-request? Sébastien made a suggestion a week ago, but you have not yet answered.

This PR has been tested successfully in our `integration` branch, but I cannot merge it into CGAL if there is an ongoing suggestion.

Reply:
I was busy with private life, so i did not look in here.
I don't see any problem with the suggestion (and it seems this is already marked as resolved).

Reply:
Right. Sébastien already did it in https://github.com/CGAL/cgal/pull/5295/commits/c30056b126f794d3fb2e9379a5a33162a6e042fd.